src/components/login-form.tsx

```diff
@@ -1,73 +1,136 @@
 import { zodResolver } from '@hookform/resolvers/zod';
-import type { SubmitHandler } from 'react-hook-form';
+import images from 'assets';
+import { ImageBackground as RNImageBackground } from 'expo-image';
+import { Link } from 'expo-router';
+import { cssInterop } from 'nativewind';
 import { useForm } from 'react-hook-form';
 import { KeyboardAvoidingView } from 'react-native-keyboard-controller';
-import z from 'zod';
 
-import { Button, ControlledInput, Text, View } from '@/ui';
+import {
+  type InputConstantsProps,
+  type InputProps,
+  type InputVariableProps,
+  type LoginFormProps,
+  type LoginFormType,
+} from '@/types/auth/auth-types';
+import { loginSchema } from '@/types/auth/auth-types';
+import { Button, ControlledInput, Image, Text, View } from '@/ui';
 
-const schema = z.object({
-  name: z.string().optional(),
-  email: z
-    .string({
-      required_error: 'Email is required',
-    })
-    .email('Invalid email format'),
-  password: z
-    .string({
-      required_error: 'Password is required',
-    })
-    .min(6, 'Password must be at least 6 characters'),
-});
+const ImageBackground = cssInterop(RNImageBackground, { className: 'style' });
 
-export type FormType = z.infer<typeof schema>;
+const TopImageLogo = () => {
```

Comment:
This component is the same as the one defined in the sign form.
Consider moving reusable components to their file definition and reuse them from there.